tests/unit/backend/models/ExportModelTest.php

```diff
@@ -50,6 +50,11 @@ public function testEncodeArrayValue()
 
     public function testDownload()
     {
+        // MIME detection is improved in PHP8
+        $expectedContentType = PHP_VERSION_ID >= 80000 && extension_loaded('fileinfo')
```

Comment:
MIME detection is improved in PHP8 (fileinfo extension), so csv is now detected correctly, not text/plain as before

Reply:
@tolkamsoft I'd be happy for the expected content type to be `application/csv` or `text/plain`, irrespective of PHP version. The MIME detection can be improved in PHP 7 in other ways, so it's possible this is already happening for people. Would you be willing to refactor this test slightly to accept either content types as acceptable without the checks for PHP8 or `fileinfo`?

Reply:
Actually I'm not sure that testing the detected content type is meaningful. AFAIK this depends on server configuration (uses magic.mime file to detect mime type). So I'd prefer not to test for it at all.

Reply:
@tolkamsoft You're correct, but most of the time, people use the bundled magic database in PHP, so it's reasonable enough to assume that this test will pass in the majority of cases. The test is there so it doesn't come out in some wacky format like `video/mpeg4` or something else very wrong.